### PR TITLE
[angkor] Minor doc fix

### PR DESCRIPTION
**Pull request by pfusik**
This corrects a typo and formatting.

ONE-DCO-1.0-Signed-off-by: Piotr Fusik <p.fusik@samsung.com>

### CODE REVIEW

**Comment by pfusik** [comment]
Splitting #12695 to multiple PRs as requested by @seanshpark

File: compiler/angkor/README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 ## Purpose
 
-_angkor_ is a `nncc` core library
+_angkor_ is an `nncc` core library
```
Comment (seanshpark) [inline compiler/angkor/README.md:5]: why `an` ?

Comment (seanshpark) [reply compiler/angkor/README.md:5]: found that `nncc` is pronounced like `en en see see` so it would be better with `an` , not `a`

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/angkor/README.md (hunk 1 above)
Comment (pfusik) [reply compiler/angkor/README.md:5]: Yes, that's exactly the rule for "a" vs "an" in English. It only matters how you pronounce the following word, not how you spell it.
For example: an XML ("eks em el") vs a XAML ("a zammel").

**Pull request merged**